<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWDialogLocate</class>
 <widget class="QDialog" name="QWDialogLocate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>单元格定位与文字设置</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>列  号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinColumn"/>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkBoxColumn">
        <property name="text">
         <string>列增</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>行  号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinRow"/>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="chkBoxRow">
        <property name="text">
         <string>行增</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>设定文字</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="editCellText"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnSetText">
        <property name="text">
         <string>设定文字</string>
        </property>
        <property name="icon">
         <iconset resource="../Resource/res.qrc">
          <normaloff>:/icons/images/322.bmp</normaloff>:/icons/images/322.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>关  闭</string>
        </property>
        <property name="icon">
         <iconset resource="../Resource/res.qrc">
          <normaloff>:/icons/images/132.bmp</normaloff>:/icons/images/132.bmp</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>QWDialogLocate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
